spec/features/user/logged_in_user_can_logout_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature 'a user who is logged in' do
+  before(:each) do
+  @user = User.create(email: "what@whatwhat.com",
+                      password: "Seekrit",
+                      full_name: "Sir Whats",
+                      display_name: "mrwhatsalot")
+  visit root_path
+  click_on "Login"
+
+  fill_in "Email", with: @user.email
+  fill_in "Password", with: "Seekrit"
+
+  click_button "Login"
+  end
+  
+  scenario 'can logout' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.